code/controllers/subsystem/statpanels.dm

```diff
@@ -0,0 +1,257 @@
+SUBSYSTEM_DEF(statpanels)
+	name = "Stat Panels"
+	wait = 4
+	init_order = INIT_ORDER_STATPANELS
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+	var/list/currentrun = list()
+	var/encoded_global_data
+	var/mc_data_encoded
+	var/list/cached_images = list()
```

Comment:
autodoc if you can (aka if you know what they are doing)

If you don't, just ignore that comment